common4j/src/main/com/microsoft/identity/common/java/broker/CommonRefreshTokenCredentialProvider.kt

```diff
@@ -49,4 +49,13 @@ object CommonRefreshTokenCredentialProvider : IRefreshTokenCredentialProvider {
         Logger.warn(methodTag, "mRefreshTokenCredentialHolder is not initialized!")
         return null
     }
+
+    override fun getRefreshTokenCredential(inputUrl : String, username : String) : String? {
+        val methodTag = "$TAG:getRefreshTokenCredential";
```

Comment:
nit: use safe call instead of not null assertion 
return mRefreshTokenCredentialProvider?.getRefreshTokenCredential(inputUrl, username)